src/Command/DrupalUpdate/DrupalUpdateCommand.php

```diff
@@ -0,0 +1,105 @@
+<?php
+namespace Acquia\Cli\Command\DrupalUpdate;
+
+use Acquia\Cli\Command\CommandBase;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DrupalUpdateCommand extends  CommandBase
+{
+  /**
+   * @var string
+   */
+  protected static $defaultName = 'app:update-d7-packages';
+  /**
+   * @var mixed
+   */
+  private $drupalRootPath;
+  /**
+   * @var mixed
+   */
+  private  $drupalCoreVersion;
+
+  /**
+   * @return mixed
+   */
+  public function getDrupalRootPath() {
+    return $this->drupalRootPath;
+  }
+
+  /**
+   * @param mixed $drupalRootPath
+   */
+  public function setDrupalRootPath($drupalRootPath): void {
+    $this->drupalRootPath = $drupalRootPath;
+  }
+
+  /**
+   * @return mixed
+   */
+  public function getDrupalCoreVersion() {
+    return $this->drupalCoreVersion;
+  }
+
+  /**
+   * @param mixed $drupalCoreVersion
+   */
+  public function setDrupalCoreVersion($drupalCoreVersion): void {
+    $this->drupalCoreVersion = $drupalCoreVersion;
+  }
+
+  /**
+   * {inheritdoc}.
+   */
+  protected function configure() {
+    $this->setDescription('Updates modules, themes, and distributions for a Drupal 7 application.')
+            ->setHidden(TRUE);
+  }
+
+  /**
+   * @param \Symfony\Component\Console\Input\InputInterface $input
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   *
+   * @return int
+   * @throws \Acquia\Cli\Exception\AcquiaCliException
+   */
+  protected function execute(InputInterface $input, OutputInterface $output) {
+    $this->validateCwdIsValidDrupalProject();
+    $this->io->note('Start checking of available updates.');
+    $this->setDrupalRootPath(getcwd());
+    $drupal_root_path = $this->getDrupalRootPath();
+    $this->determineCorePackageVersion($drupal_root_path);
+    $check_package_info = new CheckPackageInfo($input, $output);
```

Comment:
The `CheckPackageInfo` should be named as a noun, not with a verb. I'd expect this to be called something like `DrupalPackageInfo` or `DrupalPackage` if I understand it correctly.